components/gitpod/gitpod.libsonnet

```
@@ -951,6 +951,32 @@ function(params) {
     },
   },
 
+  workspacePodMonitor: {
+    apiVersion: 'monitoring.coreos.com/v1',
+    kind: 'PodMonitor',
+    metadata: {
+      name: 'workspace',
+      namespace: $._config.namespace,
+      labels: $._config.commonLabels,
+    },
+    spec: {
+      selector: {
+        matchLabels: {
+          'component': 'workspace',
```

```suggestion
          component: 'workspace',
          workspaceType: 'regular',
```

I believe this would be enough to ignore prebuilds 🙂